pkgs/development/compilers/c3c/default.nix

```diff
@@ -37,6 +46,15 @@ llvmPackages.stdenv.mkDerivation rec {
     cp -r lib $out
   '';
```

Comment:
I know that this wasn't caused by your change, nevertheless we should keep the pre-/post-phase hooks for `checkPhase` and `installPhase`. See https://nix.dev/tutorials/packaging-existing-software#phases-and-hooks

Reply:
Thanks! Added pre/post phases to checkPhase. Turns out, we do not need any modifications to the installPhase so I removed it.